### ISSUE TITLE
Declare if extension is safe for parallel reading

### ISSUE THREAD

**Issue by explode** [piccolo-orm/piccolo_theme#73]
Doing a Sphinx build using [multiple processes](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-j) will complain:

    WARNING: the piccolo_theme extension does not declare if it is safe for parallel reading, 
    assuming it isn't - please ask the extension author to check and make it explicit
    WARNING: doing serial read

[setup(app)](https://github.com/piccolo-orm/piccolo_theme/blob/61a1256b6c985eb019fabbd65512212895459dda/piccolo_theme/__init__.py#L11) probably needs to ``return {'parallel_read_safe':True}`` but would it be set True or False?

[Sphinx API docs](https://www.sphinx-doc.org/en/master/extdev/index.html#extension-metadata)

**Comment by dantownsend** [comment]
Sounds straight forward enough. I'll create a PR.

**Comment by dantownsend** [comment]
@explode Is it fixed for you in `piccolo-theme==0.21.0`?

**Comment by explode** [comment]
Yes, thanks.  You may be interested, we benchmarked HTML build [time](https://man7.org/linux/man-pages/man1/time.1.html) for some themes against a massive Sphinx project... then ditched ``furo``:

    theme                real      user      sys         cat /proc/loadavg (1m 5m 15m)
    pydata_sphinx_theme  1m08s     3m20s     0m6.988s    2.54 3.50 2.67  
    sphinx_rtd_theme     1m08s     3m42s     0m7.212s    2.63 3.22 2.65 
    piccolo_theme        1m09s     3m46s     0m7.197s    3.35 4.05 2.73  
    alabaster            1m10s     3m37s     0m7.536s    2.04 1.30 1.20  
    furo                 2m44s    16m17s    0m13.319s    5.94 4.54 2.72  
    bootstrap            3m49s    17m55s    0m15.222s    4.44 3.19 1.97

**Comment by dantownsend** [comment]
That's interesting to know - thanks. I'm not sure why some of the other themes are so much slower.
